cosmic-settings/src/pages/display/mod.rs

```diff
@@ -1098,6 +1127,43 @@ pub fn display_configuration() -> Section<crate::pages::Message> {
         })
 }
 
+pub fn legacy_applications() -> Section<crate::pages::Message> {
+    let mut descriptions = Slab::new();
+
+    let system = descriptions.insert(fl!("legacy-applications", "scaled-by-system"));
+    let system_desc = descriptions.insert(fl!("legacy-applications", "system-description"));
+    let native = descriptions.insert(fl!("legacy-applications", "scaled-natively"));
+    let native_desc = descriptions.insert(fl!("legacy-applications", "native-description"));
+
+    Section::default()
+        .title(fl!("legacy-applications"))
+        .descriptions(descriptions)
+        .view::<Page>(move |_binder, page, section| {
+            let descriptions = &section.descriptions;
+            widget::settings::view_section(&section.title)
+                .add(widget::settings::item_row(vec![widget::radio(
+                    widget::column()
+                        .push(text::heading(&descriptions[system]))
+                        .push(text(&descriptions[system_desc])),
```

Comment:
Looking at the designs, this should probably be `text::body` and `text::caption` for system and system_desc (and native/native_desc), respectively.

Reply:
Totally right, thanks for pointing that out.